library/Terminal42/ChangeLanguage/Helper/LabelCallback.php

```diff
@@ -35,7 +35,9 @@ public function register($table, callable $callback)
             $args = \func_get_args();
             $result = null;
 
-            if (\is_callable($this->previous)) {
+            $isSymfonyCallback = version_compare(VERSION, '4.0', '>=') && is_array($this->previous) && \Contao\System::getContainer()->has($this->previous[0]);
+
+            if (\is_callable($this->previous) || $isSymfonyCallback) {
```

Comment:
Wouldn't just a check for `is_array` be sufficient? It would then call `System::importStatic()` which will throw an exception if the callback is invalid. I think the idea of the if-condition was that we don't know if a callback was set at all.

Reply:
OK, that would also be allright I think. I just added it to the PR.